draft-nbtjjl-dnsop-prefer8781.md

```diff
@@ -1,9 +1,9 @@
 ---
-title: "Deprecation of DNS64 for Discovery of IPv6 Prefix Used for IPv6 Address Synthesis"
-abbrev: "Deprecate RFC7050"
+title: "Preference of pref64 over DNS64 for Discovery of IPv6 Prefix Used for IPv6 Address Synthesis"
+abbrev: "Prefer RFC8781"
```

Comment:
Maybe we shall phrase it as 'discouraging using RFC7050' - that way it would be:
- more in scope of dnsops
- stay generic (in the unlikely event of a new NAT64 prefix detection method is invented, or people want to use PCP)

WDYT?